owner-onboarding-server/src/main.rs

```diff
@@ -266,10 +423,18 @@ async fn main() -> Result<()> {
         handlers::done,
     );
 
+    // TODO(runcom): enable this conditionally from configuration
```

Comment:
It's significantly easier to instead always add the handler, but make the route itself just return an error if it is disabled (and make it disabled by default).
It's... tricky to create a `let routes =` where sometimes a route is there and sometimes it isn't.

Reply:
For an example here, see the DIUN implementation.

Reply:
I’ve pushed again by just passing a Boolean that I’ll later set - I’ll revisit this PR again tomorrow morning

Reply:
Right looking at the DIUN example I’m doing mostly the same except I’m returning 404 directly - no need to expose this at all even if it’s disabled